externals/header.phtml

```diff
@@ -34,12 +34,12 @@
     <meta http-equiv="content-type" content="text/html; charset=UTF-8" />
 
     <!--meta properties for facebook Open Graph-->
-    <meta property="og:title" content="Free Accounting Software | GnuCash"/>
+    <meta property="og:title" content="<?=T_("Free Accounting Software")?> | GnuCash"/>
     <meta property="og:type" content="website"/>
     <meta property="og:url" content="https://www.gnucash.org/"/>
     <meta property="og:image" content="https://www.gnucash.org/externals/logo_w120.png"/>
     <meta property="og:site_name" content="GnuCash"/>
-    <meta property="og:description" content="A personal and small-business financial-accounting software, licensed under GNU/GPL and available for Linux, Windows, Mac OS X, BSD, and Solaris."/>
+    <meta property="og:description" content="<?=T_("<span class="gnucash">GnuCash</span> is personal and small-business financial-accounting software, freely licensed under the <a href="https://www.gnu.org/">GNU</a> GPL and available for GNU/Linux, BSD, Solaris, Mac OS X and Microsoft Windows.")?>"/>
```

Comment:
In order to avoid "Gnu" and "Microsoft", as well as to add "double entry accounting" into the meta-tag Description, my proposal would be:
`<span class="gnucash">GnuCash</span> is a personal and small-business financial software with double entry accounting, licensed under the <a href="https://www.gnu.org/">GPL</a> and freely available for Linux, BSD, Solaris, Mac OS X and MS Windows.
`
This could be done either by changing the current original string in the po file (#: index.phtml:38), or by adding a new string to the po file. In both cases translations would need to be updated.

BTW, separate _descriptions_ tags in the <head> area of html files are nowadays not recommended any more.

Reply:
Alternative: 
instead of "… freely available for …", it could be "… a free personal …"

`<span class="gnucash">GnuCash</span> is a free personal and small-business financial software with double entry accounting, licensed under the <a href="https://www.gnu.org/">GPL</a> and available for Linux, BSD, Solaris, Mac OS X and MS Windows.`

Reply:
The standard term is "Free Open Source" or sometimes "Free/Libre Open Source". Capitalize Free because it's Free as in Freedom as opposed to free as in free beer.
The core of GnuCash is double-entry accounting and the target user population is personal and (very) small business.

Reply:
I have mixed feelings about the name GNU/Linux, I think it's RMS taking credit for more than GNU is due. 
OTOH it should be "Apple MacOS™" (it's not "Mac OS X" anymore) and "Microsoft Windows™", those are the registered trademark names of the operating systems.

Reply:
Linuux too: https://www.linuxfoundation.org/trademark-usage/
More about the story behind gnu vs. linux: https://www.linuxtoday.com/infrastructure/2007091401526opll